modules/acs-cloud-requirements.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acscs-default-requirements.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-requirements_{context}"]
+= General requirements for RHACS Cloud Service
+
+[role="_abstract"]
+{product-title-managed} has some system requirements that must be met before you can install it.
+
+[WARNING]
+====
+You must not install {product-title-managed-short} on:
+
+* Amazon Elastic File System (Amazon EFS). Use the Amazon Elastic Block Store (Amazon EBS) with the default *gp2* volume type instead.
+* Older CPUs that do not have the Streaming SIMD Extensions (SSE) 4.2 instruction set.
+For example, Intel processors older than _Sandy Bridge_ and AMD processors older than _Bulldozer_.
+(These processors were released in 2011.)
+====
+
+To install {product-title-managed-short}, you must have one of the following systems:
+
+* {ocp} version {ocp-supported-version} or later, and cluster nodes with a supported operating system of {op-system-first} or {op-system-base-full}.
+* a supported managed Kubernetes platform, and cluster nodes with a supported operating system of Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
```

Comment:
+1 to Audrey's comment, but I'd also remove the periods at the end of each line, [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists). (Alternatively, you can make them into complete sentences.)

> Write lists so that all or none of the items start with complete sentences. If list items comprise only complete sentences, include a period after each sentence. Except in definition lists, do not include any end punctuation if list items comprise only sentence fragments.
>
> If a list contains list items that start with sentence fragments, and one or more of the fragments are followed by complete sentences, include periods after all sentence fragments and sentences.